packages/cli/templates/vue3/src/components/FieldPlugin.vue

```diff
@@ -13,6 +13,7 @@ function closeModal() {
 
 <template>
   <div>
+    <!-- #region DELETE THIS BOILERPLATE -->
```

Comment:
We could

- extract these lines into a component
- move that component and all the dependencies into a new directory `src/components/FieldPluginExample`

In this way, the user can just delete this entire folder + 2 lines of code (the import statement + the place in the  template/render function where it's being used)

Reply:
If we extract these lines into a component, then it's basically the same as the whole FieldPlugin.vue. Then what if we just rename FieldPlugin.vue to FieldPluginExample/? Did you mean to still keep FieldPlugin.vue as a container for `loaded` slot?

Reply:
Oh, true! Yeah, there's no point in creating another component 😉 Renaming `FieldPlugin` -> `FieldPluginExample` would be sufficient.